htdocs/install/mysql/tables/llx_element_tag.key.sql

```diff
@@ -0,0 +1,23 @@
+-- ============================================================================
+-- Copyright (C) 2021 Maxime Kohlhaas       <support@atm-consulting.fr>
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see <https://www.gnu.org/licenses/>.
+--
+-- ============================================================================
+
+ALTER TABLE llx_element_tag ADD PRIMARY KEY pk_element_tag (fk_categorie, fk_element);
+ALTER TABLE llx_element_tag ADD INDEX idx_element_tag_fk_categorie (fk_categorie);
```

Comment:
Because you already have an index on couple   fk_categorie, fk_element, an index on fk_categorie alone is useless (fk_categorie is already the begining of index on the couple)